src/alpaca_eval/models_configs/internlm2-chat-20b-ppo/configs.yaml

```diff
@@ -0,0 +1,15 @@
+internlm2-chat-20b-ppo:
+  prompt_template: "internlm2-chat-20b-ppo/prompt.txt"
+  fn_completions: "huggingface_local_completions"
+  completions_kwargs:
+    model_name: "internlm/internlm2-chat-20b"
+    model_kwargs:
+      torch_dtype: "bfloat16"
+      trust_remote_code: True
+    is_fast_tokenizer: False
+    max_new_tokens: 2048
```

Comment:
note that we typically use 4096 for newer models. 2048 is perfectly fine just saying in case some outputs are truncated

Reply:
Thanks for your advice!

Currently we haven't observed too many truncated completions. We suppose `2048` tokens is enough for our model. Responses above `2048` tokens are generally nonsense repetition, which might waste GPU time & GPT tokens